OpenLaTeXStudio/EditorModule/src/latexstudio/editor/toolbar/ToggleComment.java

```diff
@@ -0,0 +1,46 @@
+/* 
+ * Copyright (c) 2015 Sebastian Brudzinski
+ * 
+ * See the file LICENSE for copying permission.
+ */
+package latexstudio.editor.toolbar;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import latexstudio.editor.EditorTopComponent;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.NbBundle.Messages;
+import org.openide.windows.TopComponent;
+import org.openide.windows.WindowManager;
+
+/**
+ *
+ * @author Labelpm
+ */
+@ActionID(
+        category = "Edit",
+        id = "latexstudio.editor.ToggleComment"
+)
+@ActionRegistration(
+        iconBase = "latexstudio/editor/resources/icons/comment_symbol.png",

```

Comment:
That's correct. But in the pull request there are 3 of them. One named comment-symbol24 and one comment_symbol24. The first one is not required (since the original file is named comment_symbol)